airbyte-integrations/connectors/source-zendesk-support/unit_tests/integrations/zs_responses/pagination_strategies/cursor_based_pagination_strategy.py

```diff
@@ -1,13 +1,22 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 
-from typing import Any, Dict
+from typing import Any, Dict, Optional
 
 from airbyte_cdk.test.mock_http.response_builder import PaginationStrategy
 
 
 class CursorBasedPaginationStrategy(PaginationStrategy):
-    @staticmethod
-    def update(response: Dict[str, Any]) -> None:
+    def __init__(self, first_url: Optional[str] = None) -> None:
+        self._first_url = first_url
+
+    def update(self, response: Dict[str, Any]) -> None:
+        """
+        Only allow for one page
+        """
         response["meta"]["has_more"] = True
         response["meta"]["after_cursor"] = "after-cursor"
         response["meta"]["before_cursor"] = "before-cursor"
+        if self._first_url:
```

Comment:
The low-code version implemented a `links_next_paginator` that relies on `links.next` and not on `meta.after_cursor` hence I updated this strategy to support both